packages/ui/src/components/Meter/Meter.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled'
+import type { JSX } from 'react'
+import { Text } from '../Text'
+
+const StyledTitle = styled(Text)`
+  display: inline-block;
+  vertical-align: top;
+  line-height: 22px;
+`
+
+const StyledStrength = styled(Text, {
+  shouldForwardProp: prop => !['strength'].includes(prop),
+})<{ strength: Strength }>`
+  float: right;
+  vertical-align: top;
+  color: ${({ strength }) => strength.color};
+`
+
+const StyledWrapper = styled.div`
+  background-color: ${({ theme }) => theme.colors.neutral.backgroundDisabled};
+  border-radius: ${({ theme }) => theme.radii.default};
+  height: ${({ theme }) => theme.space['1']};
+  margin-top: ${({ theme }) => theme.space['1']};
+  margin-bottom: ${({ theme }) => theme.space['2']};
+`
+
+const StyledMeter = styled.div`
+  border-radius: ${({ theme }) => theme.radii.default};
+  height: 100%;
+  transition: all 0.5s;
+`
+
+type Strength = {
+  /**
+   * Color to display
+   */
+  color: string
+  /**
+   * Text to display
+   */
+  text: string
+}
+
+type PasswordStrengthMeterProps = {
+  /**
+   * Strength is used for defining different color and text associated with it.
+   */
+  strength: Strength[]
+  title: string
+  value: number
+  className?: string
+  'data-testid'?: string
+  id?: string
+}
+
+export const Meter = ({
+  strength,
+  title,
+  value,
+  className,
+  'data-testid': dataTestId,
+  id,
+}: PasswordStrengthMeterProps): JSX.Element => {
+  const toValue = ((value + 1) / strength.length) * 100
+  const width = `${toValue}%`
+
+  const backgroundColor = strength[value]?.color ?? strength[0].color
+  const text = strength[value]?.text ?? strength[0].text
+
+  return (
+    <div
+      aria-live="polite"
+      aria-valuemax={strength.length ?? 0}
+      aria-valuemin={0}
+      aria-valuenow={value}
+      className={className}
+      data-testid={dataTestId}
+      id={id}
+      role="progressbar"
```

Comment:
Why not directly using tag `meter` in html and using the native props? I see you used aria-valuemin/max etc those could be the native props

Reply:
The meter element is a complex element to style. We have 2 options in this case: 
- We can hide visually the meter and use our own design
- We can try to style the meter but we will need to check with the Design Team because it's will be impossible to be 100% as the current design.

Or we can stay with a div that respect meter pattern ?

@matthprost

Reply:
@fabienhebert @philibea we can't unify both components as there have different behavior and usages. But your comment makes me think we should change `role="progressbar"` to `role="meter"` instead